TEXTBOOK_README.md

```diff
@@ -0,0 +1,113 @@
+6. Join project [Qiskit/platypus] and then visit the language you signed up to contribute to.<br/>
+7. For questions about the project, to connect with other translators, or to receive updates regarding the project, please join #qiskit-localization channel in the [Qiskit workspace](http://ibm.co/joinqiskitslack). <br/>
+
+## Review the CLA document
+The [Qiksit Contribution License Agreement (CLA)](https://qiskit.org/license/qiskit-cla.pdf) document is available for review as PDF.
+
+## What is GitLocalize?
+
+[GitLocalize](https://gitlocalize.com) is a platform for managing translations of the content stored on GitHub and simplifies the translation workflow for developers.
+
+Important note:<br/>
+Gitlocalize is under active development, so you may encounter issues during translation. If you find bugs or errors, please feel free to reach out to translation leads or open an issue [here](https://github.com/gitlocalize/gitlocalize-ibm/issues). 
+
+
+## How to add a new language?
+
+The Qiskit Localization Project currently supports: Bengali, Chinese Simplified, Chinese Traditional, French, German, Greek, Hindi, Hungarian, Indonesian, Italian, Japanese, Korean, Malayalam, Marathi, Polish, Portuguese, Russian, Spanish, Swahili, Tamil, Thai, Turkish, Urdu, and Vietnamese. 
+
+If you want to add a new language and become a translation lead, you can open a [GitHub issue](https://github.com/qiskit-community/qiskit-translations/issues/new/choose) to start a discussion with the Qiskit team and recruit translation project members. Please refer to the [criteria](#what-is-the-criteria-for-adding-a-new-language?) below to receive official support from the administrators for new languages.
+
+
+## What is the criteria for adding a new language?
+
+1. A minimum of **three contributors** is necessary for any new language to be added and receive official support from the administrators of the Qiskit Textbook localization project.<br/>
+2. In addition to translators, we will need **dedicated proof-readers** to review the translations and approve accuracy of content in that language, to ensure the translations can be released in that language.<br/>
+3. Among the group of contributors, a **translation lead must be identified** to serve as a liaison with the administrators of the localization project. The lead must contact [Soolu Thomas](mailto:soolu.thomas@ibm.com?subject=[GitHub]%20Qiskit%20Documentation%20Translation) and [Yuri Kobayashi](mailto:yurik@jp.ibm.com?subject=[GitHub]%20Qiskit%20Documentation%20Translation) by email for the language to be officially listed under this project. <br/>
+
+These are general guidelines for joining translation efforts and adding new languages. If you have further questions, please feel free to contact [Soolu Thomas](mailto:soolu.thomas@ibm.com?subject=[GitHub]%20Qiskit%20Documentation%20Translation) and [Yuri Kobayashi](mailto:yurik@jp.ibm.com?subject=[GitHub]%20Qiskit%20Documentation%20Translation). Thank you.
```

Comment:
Maybe add a text like

```
Please note that the above does not guarantee that the language will be opened. (?)  
```

Reply:
Yes, I think this is a good suggestion. Will add.